app/models/pipeline_run.rb

```diff
@@ -11,6 +11,7 @@ class PipelineRun < ApplicationRecord
   accepts_nested_attributes_for :pipeline_run_stages
   has_and_belongs_to_many :backgrounds
   has_and_belongs_to_many :phylo_trees
+  has_and_belongs_to_many :bulk_downloads
```

Comment:
I had forgotten this line earlier. If you don't have this, you can't destroy pipeline_runs (the foreign keys in bulk_downloads_pipeline_runs table will complain)